docs/sources/features/datasources/cloudwatch.md

```diff
@@ -110,6 +111,11 @@ aws_secret_access_key = dasdasdsadasdasdasdsa
 region = us-west-2
 ```
 
+## Using the Query Editor
+CloudWatch datasource can query data from both CloudWatch metrics and CloudWatch logs APIs and each has its own specialized query editor. You select which API you want to query with using the query mode switch on top of the editor.
```

Comment:
Per the Documentation style guide, "data source" is two words. Please apply throughout this topic.